packages/babel-plugin-remove-graphql-queries/src/index.ts

```diff
@@ -139,23 +206,23 @@ function removeImport(tag) {
   }
 }
 
-function getGraphQLTag(path) {
-  const tag = path.get(`tag`)
+function getGraphQLTag(path: NodePath<TaggedTemplateExpression>): IGraphQLTag {
+  const tag = path.get(`tag`) as NodePath
   const isGlobal = isGlobalIdentifier(tag)
 
-  if (!isGlobal && !isGraphqlTag(tag)) return {}
+  if (!isGlobal && !isGraphqlTag(tag)) return {} as IGraphQLTag
 
   const quasis = path.node.quasi.quasis
 
   if (quasis.length !== 1) {
     throw new StringInterpolationNotAllowedError(
-      quasis[0].loc.end,
-      quasis[1].loc.start
+      quasis[0].loc?.end,
+      quasis[1].loc?.start
     )
   }
 
   const text = quasis[0].value.raw
-  const hash = murmurhash(text, `abc`)
+  const hash = murmurhash(text, 0)
```

Comment:
That's interesting. I get why we need `number` here (`murmurhash` expect seed to be number) - I would to spend bit more time on digging into it - there was likely cast going on - so I would want to use whatever string was cast into to not cause change in hash generation

Reply:
I thought this was interesting too. It looks like the only place the second argument `seed` is used in the `murmurhash` function is on line 6:

```let h: number = seed ^ str.length```

I tried treating `'abc'` as a hex value, and that doesn't produce the same result. I also tried getting the binary representation of `'abc'` and converting it to a decimal, and that didn't work either. 

I'm not sure what type of casting is going on, but I tested this line several times and it seems like `^ 'abc'` is the identity operator. Which is strange, because why include the XOR if you could just write `let h: number = str.length`?

Reply:
I just noticed that the `murmurhash` function was taken from [here](https://gist.github.com/raycmorgan/588423v), as indicated in the comment at the top of `murmur.ts`. That would explain why that line `let h: number = seed ^ str.length` is there despite there being a simpler alternative. I think this hash function is a general-use implementation of the MurmurHash2 algorithm, and it just so happens that in our case omitting the seed doesn't change anything.

Reply:
Fwiw, `^` is a bitwise op that will force coerce its arguments to a 32bit number, or `0` if it's not a number (explicitly, it will not cause `NaN`). Hence you can throw anything at it, or any other bitwise op, and always be guaranteed a 32bit number back. The coercion does not try different bases afaik.

Reply:
I double checked this, too, and I think the change of `'abc' -> 0` is fine or at least does not change the status quo.